package.json

```diff
@@ -70,7 +70,7 @@
     "mysql": "^2.11.1",
     "pg": "^5.0.0",
     "pg-hstore": "^2.3.2",
-    "sequelize": "^3.24.6",
+    "sequelize": "^4.23.2",
```

Comment:
need to test against ^3 aswell

Reply:
I've tested this patch with ^3 and ^4 and it works. Adding a dev dependencies for both versions is currently not possible with npm (https://github.com/npm/npm/issues/5499) However, it is with yarn. Would you consider using yarn?

Reply:
Also, would you accept the patch with the changes in package.json discarded for now?

Reply:
> Would you consider using yarn?

Definitely consider yarn, use it a lot privately.

> Also, would you accept the patch with the changes in package.json discarded for now?

yes